packages/ckeditor5-theme-lark/theme/ckeditor5-table/inserttable.css

```diff
@@ -23,6 +23,8 @@
 .ck .ck-insert-table-dropdown-grid-box {
 	width: var(--ck-insert-table-dropdown-box-width);
 	height: var(--ck-insert-table-dropdown-box-height);
```

Comment:
As you added `min-width` and `min-height` these two can be now removed. Regarding the styling I'll add some more information in the review comment.

Reply:
Yes, width an height are not needed.